packages/@unimodules/react-native-adapter/src/Platform.ts

```diff
@@ -1,7 +1,42 @@
 import { Platform as ReactNativePlatform } from 'react-native';
 
+import { isDOMAvailable } from './environment/browser';
+
+export type PlatformOSType =
+  | 'ios'
+  | 'android'
+  | 'web'
+  | 'native'
```

Comment:
What is `native`? 🤔 I think it's the first time I see such `Platform.OS`. Would it be used to designate SSR environments? I'm a little concerned it might be misleading as `native` is (at least in my mind) related to, like, original React Native-supported platforms, i.e. iOS and Android.

Reply:
"native" targets the same devices that `module.native.js` does. It targets Android and iOS (and iPadOS one day...?) if either of those platforms don't have more precise "module.{android,ios}.js" files.

Reply:
Ahhh, ok, I haven't noticed that `PlatformOSType` is only used for `PlatformSelect` and not for, well, `Platform.OS`. Thank you for explaining that! 🙏 

With that said, maybe (just maybe) it would be worth to rename `PlatformOSType` to something mentioning _specifier_, _filter_ so that it isn't semantically related that closely?